tests/contrib/pyramid/test_pyramid.py

```diff
@@ -38,6 +38,9 @@ def get_settings(self):
     def override_settings(self, settings):
         self.create_app(settings)
 
+class PyramidTestCase(PyramidBase):
```

Comment:
refactoring to use `PyramidBase` as a base TestCase class that includes all is needed to create a traced `pyramid` app